sklearn/metrics/tests/test_common.py

```diff
@@ -216,6 +216,7 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     "hinge_loss": hinge_loss,
 
     "brier_score_loss": brier_score_loss,
+    "neg_brier_score": brier_score_loss,
```

Comment:
you should revert all the changes in test_common.py and add the new scorer to test_score_objects.py

Reply:
maybe change brier_score_loss with neg_brier_score_loss in test_common.py

Reply:
and neg_brier_score_loss is added in the CLF_SCORERS in test_score_objects.py if that's what you mean by adding the new scorer to test_score_objects.py